packages/teleterm/src/ui/TopBar/Identity/Identity.tsx

```diff
@@ -44,3 +59,18 @@ export function Identity() {
     </>
   );
 }
+
+// Hack - for some reason xterm.js doesn't allow moving a focus to the Identity popover
+// when it is focused using element.focus(). Moreover, it looks like this solution has a benefit
+// of returning the focus to the previously focused element when popover is closed.
```

Comment:
It's been a while since I ran into that, but I noticed that sometimes my own shortcuts defined through BetterTouchTool don't work while some field in Teleterm is focused.

Just putting it out there in case we run into similar problems in the future.

Reply:
That's interesting. Does it happen in input field or in xterm area?

Reply:
Not sure where it was, it was either the old command bar or xterm.